gui/src/3D/materials/fluentButton/fluentButtonBuilder.ts

```diff
@@ -0,0 +1,45 @@
+import { Scene } from "babylonjs/scene";
+import { Mesh } from "babylonjs/Meshes/mesh";
+import { VertexData } from "babylonjs/Meshes/mesh.vertexData";
+import { Nullable } from "babylonjs/types";
+
+function CreateFluentButtonVertexData(): VertexData {

```

Comment:
Is it possible to have a mesh for this instead? Load it async the first time it is loaded? I am just thinking about the package size for non-VR scenes that don't require this. 
Unless it is used any place other than creating the mesh in the builder